### PR TITLE
add iasql formulae

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: Formula/iasql.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Iasql < Formula
+  desc "Infrastructure as SQL"
+  homepage "https://iasql.com"
+  url "https://github.com/alantech/iasql-releases/releases/download/v0.1.0/iasql-macos.tar.gz"
+  sha256 ""
```
Comment (depombo) [inline Formula/iasql.rb:5]: the script should fill this in?

**Review by dfellis — APPROVED** [review]
LGTM

File: .github/workflows/update-iasql.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: update IaSQL
+
+on:
+  workflow_dispatch:
+    inputs:
+      alan_version:
```
Comment (aguillenv) [inline .github/workflows/update-iasql.yml]:
```suggestion
      iasql_version:
```

**Review by aguillenv — APPROVED** [review]

File: Formula/iasql.rb (hunk 1 above)
Comment (aguillenv) [reply Formula/iasql.rb:5]: Yes, it should

**Pull request merged**